spec/models/hets_instance_spec.rb

```diff
@@ -1,14 +1,52 @@
 require 'spec_helper'
 
 describe HetsInstance do
-  context 'when creating a hets instance' do
-    context 'and it has a reachable uri' do
-      let(:general_version) { '0.99' }
-      let(:specific_version) { '1409043198' }
+  let(:general_version) { '0.99' }
+  let(:specific_version) { Hets.minimal_version_string }
+  let(:hets_instance) { create :local_hets_instance }
+
+  context 'when choosing a hets instance' do
+    context 'and there is no hets instance recorded' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.